_posts/2020-01-28-Tips-for-creating-open-source-projects-that-are-contributor-friendly.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: Tips-for-creating-open-source-projects-that-are-contributor-friendly
+date: 2020-01-28 06:31:01
+categories: [open source]
+author: bronzdoc
+
+---
+
+At Ombu Labs we are firm believers of open source. We use open source but also like to contribute back to the community and we have open sourced a few projects.
+
+We know that contributing to open source can be a difficult task so we have been looking for easier ways help to onboard new and past contributors.
+Here is a list of tips and tools that we believe that can help you make your open source project a more friendly space.
+
+# Documentation
+
+Having proper documentation for your project is key to make your project more friendly. Document the steps to setup the project, known issues and a guide on how to properly contribute to the project with a CONTRIBUTING.md or Adding steps in your README.md.
+
+# Label issues
+
+A nice thing that about issue trackers these days is that you can label issues with a short text to know what is about before reading the issue. With this a contributor can filter issues by its label and pick something to work on of his interest.
+
+Some nice labels for new contributors are:
+
+ * Help Wanted
+ * first-timers-only
+ * Beginner
+ * Good First Issue
+ * Easy
+
+
+# Code Of Conduct
+
+Having a code of conduct is important. Contributing to open source is more than code, there is people involved and a COC could help people feel safe and welcome by knowing there is a set of rules to maintain armony.
+
+We personally have been using [The Contributor Covenant](https://www.contributor-covenant.org/) but there are many more that you can change and adpat to your project needs.
```

Comment:
`change and adpat to your project needs` -> `change and adapt to your project's needs`